packages/mobile/src/screens/profile-screen/ProfileTabNavigator.tsx

```diff
@@ -107,6 +107,7 @@ export const ProfileTabNavigator = ({
       <CollapsibleTabNavigator
         renderHeader={renderHeader}
         animatedValue={animatedValue}
+        headerHeight={1000}
```

Comment:
maybe comment / make a good named const for 1000. hard to know "why" ya know?

Reply:
oh weird, latest commit didn't get pushed which approaches this exactly!